sys/include/net/gnrc/netdev.h

```diff
@@ -66,6 +69,19 @@ extern "C" {
  */
 #define GNRC_NETDEV_MAC_INFO_RX_STARTED         (0x0004U)
 
+#ifdef MODULE_GNRC_MAC
```

Comment:
This does not need to be in an `#ifdef`. The flag is just meaningless without it, but to avoid collisions its value should stay the same regardless of the modules inclusion.